test/e2e/config.go

```diff
@@ -6,4 +6,5 @@ var (
 	ALPINE        = "docker.io/library/alpine:latest"
 	infra         = "k8s.gcr.io/pause:3.1"
 	BB            = "docker.io/library/busybox:latest"
+	healthcheck   = "docker.io/libpod/alpine_healthcheck:latest"
```

Comment:
Can we put this on Quay with the other test images? (Did we ever finish migrating our test images to Quay? I know CRI-O did)

Reply:
no bc they dont build the right image type we need.